frontend/components/label/LabelAdder.jsx

```diff
@@ -50,7 +52,11 @@ const LabelAdder = ({ getData }) => {
             <LabelBox>
               <label htmlFor='label-name'>Label name</label>
             </LabelBox>
-            <Input placeholder='이름' value={preview.title} onChange={(e) => handleChangeTitle(e)} />
+            <Input
+              placeholder='Label Prieview'
+              value={preview.title}
+              onChange={(e) => handleChangeTitle(e)}
```

Comment:
이런 경우에는 함수만 넘겨도 될것같습니다!
```suggestion
              onChange={handleChangeTitle}
```